spec/concurrent/executor/cached_thread_pool_spec.rb

```diff
@@ -212,6 +212,8 @@ module Concurrent
               pid       = spawn RbConfig.ruby, test_file
               Process.waitpid pid
               expect($?.success?).to eq true
+            rescue Errno::ECHILD
+              # child already gone
```

Comment:
Ah! I was also fixing this :)

```ruby
            rescue Errno::ECHILD
              # the child could quit out fast enough before we try to wait for it!
```

I think we could _TODO_ move this test to `executor_service_shared` & have `pool_quits` start every type of executor service: cached & single executor

Reply:
yeah that sounds like a good idea! Would you be willing to do it?

Reply:
done